PrettyHtmlFormatter.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Grav\Plugin;
+
+class PrettyHtmlFormatter extends \Monolog\Formatter\HtmlFormatter
+{
+    /**
+     * Formats a log record.
+     * @param  array $record A record to format
+     * @return mixed The formatted record
+     */
+    public function format(array $record)
+    {
+        $template = <<<HTML
```

Comment:
I could be great to use the grav embedded twig instance to render this template ;)